backend/presentation/managers.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.db.models.signals import post_save
+
+
+class NotificationManager(models.Manager):
+    def bulk_create(self, objs, **kwargs):
+        notifications = super(NotificationManager, self).bulk_create(objs, **kwargs)
```

Comment:
```suggestion
        notifications = super().bulk_create(objs, **kwargs)
```
You can read more about `super()` [here](https://stackoverflow.com/questions/10482953/python-extending-with-using-super-python-3-vs-python-2).